scalardb-graphql-sample/README.md

```diff
@@ -1,130 +1,230 @@
-# ScalarDB GraphQL Sample
+# Create a Sample Application That Uses ScalarDB GraphQL
 
-This sample application is a simple CLI electronic money application that has the following features:
+{% capture notice--info %}
+**Note**
 
-- Charge an `amount` to a `user`
-- Pay an `amount` from a `user` to another `user`
-- Show the balance of a `user`
+ScalarDB GraphQL is now deprecated. Please see [Getting Started with ScalarDB Cluster](https://scalardb.scalar-labs.com/docs/latest/scalardb-cluster/getting-started-with-scalardb-cluster/) instead.
```

Comment:
Maybe it would be appropriate to state here: ScalarDB GraphQL Server is now deprecated, and consequently, this sample code is also deprecated. To use ScalarDB GraphQL, you need to use ScalarDB Cluster (enterprise only). For more information about ScalarDB Cluster, See [ScalarDB Cluster document]().

Also, the `scalardb-cluster` repository is private. So maybe we need to link to the doc site?

Reply:
Thank you! I agree that a bit more information would be helpful here. How about the following?:

```suggestion
ScalarDB GraphQL Server is now deprecated, and consequently, this sample code is also deprecated. To use the ScalarDB GraphQL interface, you need to use ScalarDB Cluster, which is available only in the Enterprise edition. For more information, see [ScalarDB Cluster](https://scalardb.scalar-labs.com/docs/latest/scalardb-cluster/).
```

> [!NOTE]
>
> The link in this case points to the docs site, so I think we should be OK.

Reply:
LGTM!